### PR TITLE
OD15838 Update bid.pmptier to type int

### PR DESCRIPTION
**Pull request by nfitzsimons-magnite**
Adding in

### CODE REVIEW

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 1]
```diff
@@ -541,6 +541,9 @@ message BidExt {
         // Right marketplace. Omitted for open auction.
         optional string pmptier = 10;
 
+        //OD15838 Temporary field to allow for transition period
```
Comment (eskenny) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:544]: This is a public schema, so we shouldn't refer to our internal tickets in the comments

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 2]
```diff
@@ -541,6 +541,9 @@ message BidExt {
...
+        optional int32 pmptier_int = 15;
```
Comment (eskenny) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:545]: Personally I'd say better to put this below `imptrackers` to keep field ordering consistent (as long as comment points to existing `pmptier`)

**Review by eskenny — COMMENTED** [review]
Do we intend to rename this to `pmptier` once current `pmptier` is retired?

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 3]
```diff
@@ -553,6 +553,9 @@ message BidExt {
         // DSP provided impression tracking pixels. If the request indicates a
         // secure environment via the imp.secure flag, all URLs must be secure.
         repeated string imptrackers = 14;
+
+        //Temporary field of pmptier to allow for transition period
```
Comment (eskenny) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:557]: Please a space after `//`

**Review by eskenny — COMMENTED** [review]
One minor comment, otherwise LGTM

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 4]
```diff
@@ -553,6 +553,9 @@ message BidExt {
         // DSP provided impression tracking pixels. If the request indicates a
         // secure environment via the imp.secure flag, all URLs must be secure.
         repeated string imptrackers = 14;
+
+        // Temporary field of pmptier to allow for transition period
+        optional int32 pmptier_int = 15;
```
Comment (cindyhere33) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:558]: Why mark it as temporary if that's the field we're going to use moving forward?

File: src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto [hunk 5]
```diff
@@ -553,6 +553,9 @@ message BidExt {
         // DSP provided impression tracking pixels. If the request indicates a
         // secure environment via the imp.secure flag, all URLs must be secure.
         repeated string imptrackers = 14;
+
+        // Temporary field of pmptier to allow for transition period
+        optional int32 pmptier = 15;
```
Comment (cindyhere33) [inline src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto:558]: the comment doesn't seem accurate anymore. Would be good to add the same detail as the `deprecated_pmptier` comment for this field.

**Review by eskenny — APPROVED** [review]

**Pull request merged**